kubernetes/docs/deploy-to-ibmcloud.md

```diff
@@ -0,0 +1,100 @@
+# Deploy to IBM Cloud
+
+## Create a Kubernetes cluster
+
+https://cloud.ibm.com/kubernetes/clusters
+
+![clusters](../imgs/clusters.png)
+
+Click on `shopping-cluster` to view:
+
+![shopping-cluster](../imgs/shopping-cluster.png)
+
+## Set up CI for IBM Cloud
+
+https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install
+
+## Connect to IBM Cloud with CLI
+
+If this is your first time connecting to an IBM Cloud cluster, see the
+[full setup directions](https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install).
+
+![connect-via-cli](../imgs/connect-via-cli.png)
+
+## Set up helm
+
+https://cloud.ibm.com/docs/containers?topic=containers-helm
+
+```sh
+kubectl create serviceaccount tiller -n kube-system
+kubectl create clusterrolebinding tiller --clusterrole=cluster-admin --serviceaccount=kube-system:tiller -n kube-system
+kubectl get serviceaccount -n kube-system tiller
+```
+
+## Set up IBM Cloud container registry
+
+https://cloud.ibm.com/kubernetes/registry/main/start
+
+## Upload images to IBM Cloud container registry
+
+```sh
+ibmcloud cr login
```

Comment:
Probably best to indicate this is to log into the container registry VS logging into the IBM account

```
      ibmcloud login -a cloud.ibm.com -r us-south -g default
        
        is required, but so is 

       ibmcloud cr login 
```

        The latter is to log into container registry.